src/component.tsx

```diff
@@ -78,5 +80,13 @@ export const RemoveScrollBar: React.FC<BodyScroll> = (props) => {
    */
   const gap = React.useMemo(() => getGapWidth(gapMode), [gapMode]);
 
+  React.useEffect(() => {
+    document.body.setAttribute(lockAttribute, '');
+
+    return () => {
+      document.body.removeAttribute(lockAttribute);
+    };
+  }, []);
```

Comment:
I believe this caused a regression. I have posted an issue explaining it: #57

Reply:
👍 should be a counter